test/parallel/test-fs-read-stream.js

```diff
@@ -140,9 +140,14 @@ const rangeFile = path.join(common.fixturesDir, 'x.txt');
   }));
 }
 
-assert.throws(function() {
-  fs.createReadStream(rangeFile, { start: 10, end: 2 });
-}, /"start" option must be <= "end" option/);
+common.expectsError(
+  () => {
+    fs.createReadStream(rangeFile, { start: 10, end: 2 });
```

Comment:
Just for this case I suggest validating the output message as well

Reply:
I added the message assertion